components/automate-gateway/api/iam/v2/tokens.proto

```diff
@@ -13,6 +14,26 @@ import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
 import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service Tokens {
+
+  /*
+  Create a token
+
+  Creates a token.
+  Name, ID are required.
+  Active defaults to true when not specified.
+  Value is auto-generated when not provided.
```

Comment:
Please add back this vital text that was removed from the .md file:

Note that this creates **non**-admin tokens that may then be assigned permissions via policies just like users or teams (unless you have already created policies that encompass all tokens using `tokens:*`).

You cannot create admin tokens via the REST API.
Admin tokens can only be created by specifying the `--admin` flag to this `chef-automate` sub-command:

```bash
  chef-automate iam token create <your-token-name> --admin
```